src/nvim/path.c

```diff
@@ -210,7 +210,7 @@ int path_head_length(void)
 /// @return
 ///   - True if path begins with a path head
 ///   - False otherwise
-bool is_path_head(const char_u *path)
+bool is_path_head(const char *path)
 {
 #ifdef MSWIN
   return isalpha(path[0]) && path[1] == ':';
```

Comment:
It seems that this should be `ascii_isalpha` instead

Reply:
ascii_isalpha doesn't exist.

Reply:
Actually maybe better to keep this. It is similarly used in other places.